### DISCUSSION TITLE
Hi, is it possible to make this for watchstone mods?

### DISCUSSION THREAD

**Discussion by KingOfHell94** [xanthics/poe_gen_gwennen#7] [Ideas]
i'd like to make filters for watchstones (titanium,chromium,platinum) so i can choose which mods i want and which tiers ideally and it would give me filter to roll with alts and highlight exactly what i want.

**Comment by xanthics** [comment]
Given a list of all possible mods, it is certainly possible.  I have a version for map mods here https://xanthics.github.io/poe_gen_search_string/

From my testing, you can only search on the explicit text and the item name.  Since watchstones are magical, the mod names are exposed though.

I'd thought about making a search generator for all possible crafting mods, bucketed by basetype.  But that will involve a bit of fiddling around with RePoE.
